README.md

```diff
@@ -29,41 +29,58 @@ The main module itself provides only API that helps fetch data from TractionRec.
 ### Create a Connected App
 
 1. Create a new private key and X509 certificate, customizing the `subj` options in the command to suit your organization. (See [the manual for openssl-req](https://www.openssl.org/docs/manmaster/man1/openssl-req.html) to understand the options here.)
-  ```shell
-  openssl req -x509 -noenc -sha256 -days 365 -keyout traction_rec.key -out traction_rec.crt -subj "/C=US/ST=Illinois/L=Chicago/O=My YMCA/OU=Org/emailAddress=youremail@example.com"
-  ```
+    ```shell
+    openssl req -x509 -noenc -sha256 -days 365 \
```

Comment:
We need to ensure we are covering 365 days limit in the docs and users are aware about the process